src/main/resources/StoreSchema

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS category,property,item,catalog;
+
+CREATE TABLE IF NOT EXISTS category(
+ID INT AUTO_INCREMENT PRIMARY KEY,
+title VARCHAR(100),
+description VARCHAR(20),
+UNIQUE(title,description)
+);
+
+CREATE TABLE IF NOT EXISTS property(
+ID INT PRIMARY KEY AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+value INT NOT NULL,
+UNIQUE(name, value),
+);
+
+CREATE TABLE IF NOT EXISTS catalog (
+ID INT PRIMARY KEY AUTO_INCREMENT,
+name VARCHAR(100),
+UNIQUE(name)
+);
+CREATE TABLE IF NOT EXISTS item (
+ID INT PRIMARY KEY AUTO_INCREMENT,
+title VARCHAR(100),
+description VARCHAR(20),
+price INT,
+UNIQUE(title,description,price),
+PROPERTY_ID INT,
+CATEGORY_ID INT,
+CATALOG_ID INT,
+FOREIGN KEY (PROPERTY_ID) REFERENCES property(id),
+FOREIGN KEY (CATEGORY_ID) REFERENCES category(id),
```

Comment:
From the task [README](https://github.com/ChangeRequest/jdbc.hw1/blob/master/README.md):

> One item should be related to many categories. Many items may be related to many categories.

With your implementation there is no way to link item to many categories